plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2734,8 +2734,12 @@ elementgui.potioneffect.event_tick_condition=Active tick condition
 elementgui.potioneffect.display_name=Potion effect display name: 
 elementgui.potioneffect.category=Potion effect category: 
 elementgui.potioneffect.color=Potion effect color: 
+elementgui.potioneffect.cures=Potion cures: 
 elementgui.potioneffect.icon=Effect icon
 elementgui.potioneffect.instant=<html>Is this potion effect instantly applied?<br><small>If this is checked, potion effect tick procedure will not be called
+elementgui.potioneffect.is_cured_by_honey = Honey
```

Comment:
Why space between and after = ? This is not how this file is formatted...

Reply:
Fixed